scripts/gen_manifests.sh

```diff
@@ -50,6 +45,11 @@ SONATAFLOW_CR=manifests/01-sonataflow_${workflow_id}.yaml
 yq --inplace eval '.metadata.annotations["sonataflow.org/profile"] = "prod"' "${SONATAFLOW_CR}"
 
 yq --inplace ".spec.podTemplate.container.image=\"quay.io/orchestrator/serverless-workflow-${workflow_id}:latest\"" "${SONATAFLOW_CR}"
+yq --inplace ".spec.podTemplate.container.envFrom=[{\"secretRef\": { \"name\": \"${workflow_id}-creds\"}}]" "${SONATAFLOW_CR}"
+
+if [ "$ENABLE_PERSISTENCE" = false ]; then
```

Comment:
I'm handling this as part of #313

Reply:
I was hoping for your pr to be merged first and have the auto merge do its job
But I'll revert this change